### PR TITLE
Stringtable: Localise STR_NOTF_BlackListedVehicle

### PR DESCRIPTION
**Pull request by Jawshy**
@DardoTheMaster, you never added this to the stringtable in your pull
request https://github.com/ArmaLife/Framework/pull/532 / https://github.com/ArmaLife/Framework/commit/14692a00d8977af73ea16b8fb8dc07e03d73e0bf.

Edit: 
Thanks @DardoTheMaster. String localised. Could fluent foreign language speakers please verify the accuracy of the translations.

### CODE REVIEW

File: Altis_Life.Altis/stringtable.xml [hunk 1]
```diff
@@ -4175,6 +4175,17 @@
...
+        <Key ID="STR_NOTF_BlackListedVehicle">
+            <Original>%2 got back a blacklisted vehicle near %1.</Original>
+            <Czech>%2 dostat zpět na černou listinu vozidlo nedaleko %1.</Czech>
+            <French>%2 revenir un véhicule près de la liste noire %1.</French>
+            <Spanish>%2 volver un vehículo en la lista negra cerca de %1.</Spanish>
+            <Italian>%2 tornare un veicolo nella lista nera nei pressi %1.</Italian>
+            <Polish>%2 wrócić do czarnej listy pojazd pobliżu %1.</Polish>
+            <Portuguese>%2 voltar um veículo na lista negra perto %1.</Portuguese>
+            <Russian>%2 получить обратно в черный список автомобиль возле %1.</Russian>
+            <German>%2 erhalten eine schwarze Liste gesetzt Fahrzeug in der Nähe %1 zurück.</German>
```
Comment (setoy) [inline Altis_Life.Altis/stringtable.xml]: German: %2 hat ein gesuchtes Fahrzeug Nähe %1 ausgeparkt.

Comment (Jawshy) [inline Altis_Life.Altis/stringtable.xml]: Thanks, @setoy. I'll commit the fix to the pull once I get a few more corrections.

File: Altis_Life.Altis/stringtable.xml [hunk 2]
```diff
@@ -4175,6 +4175,17 @@
             <Russian>Вы не можете удалить элемент, когда вы находитесь в автомобиле.</Russian>
             <German>Sie können ein Element nicht entfernen, wenn Sie in einem Fahrzeug sind.</German>
         </Key>
+        <Key ID="STR_NOTF_BlackListedVehicle">
+            <Original>%2 got back a blacklisted vehicle near %1.</Original>
+            <Czech>%2 dostat zpět na černou listinu vozidlo nedaleko %1.</Czech>
+            <French>%2 revenir un véhicule près de la liste noire %1.</French>
+            <Spanish>%2 volver un vehículo en la lista negra cerca de %1.</Spanish>
```
Comment (Jawshy) [inline Altis_Life.Altis/stringtable.xml]: Hi @Shield1739, could you check that the Spanish translation is accurate?

**Comment by danielstuart14** [comment]
<Portuguese>%2 retirou um veículo que se encontra na lista negra, perto de %1.</Portuguese>

File: Altis_Life.Altis/stringtable.xml [hunk 3]
```diff
@@ -4423,16 +4423,16 @@
             <Portuguese>Multando %1</Portuguese>
             <Polish>Mandat dla %1</Polish>
         </Key>
-        <Key ID="STR_COP_BlackListedVehicle">
+        <Key ID="STR_NOTF_BlackListedVehicle">
             <Original>%2 got back a blacklisted vehicle near %1.</Original>
-            <Czech>%2 dostat zpět na černou listinu vozidlo nedaleko%1.</Czech>
+            <Czech>%2 dostat zpět na černou listinu vozidlo nedaleko %1.</Czech>
             <French>%2 revenir un véhicule près de la liste noire %1.</French>
```
Comment (Jawshy) [inline Altis_Life.Altis/stringtable.xml]: @NiiRoZz, could you check that the French translation is accurate?

**Pull request merged**